api/src/main/scala/org/virtuslab/ideprobe/ProbeExtensions.scala

```diff
@@ -111,7 +111,7 @@ trait ProbeExtensions {
     def makeExecutable(): Path = {
       import java.nio.file.attribute.PosixFilePermission._
       val attributes = Files.getPosixFilePermissions(path)
-      attributes.add(OWNER_EXECUTE)
+      attributes.add(OTHERS_EXECUTE)
```

Comment:
this is like changing `chmod u+x` to `chmod o+x` while we want `chmod +x` so we need 3 .add calls actually